### PR TITLE
Fixes #1272

### LINKED ISSUE #1272: App Crashes
**Summary:**

'Pay To' field accepts empty entries and the spinner is empty which is not good at all.
'Pay From' field accepts empty entries and the spinner is empty which is not good at all.

App crashes,
On following actions:
Transfer->Transfer->Continue->Continue->App Crashed

**Expected behaviour:**

Give an error after pressing continue if the field was empty and if it succeeds,
Ask for Amount

**Observed behaviour:**

Desktop_20200118233021

**Device and Android version:**

One Plus 6
Android Version- Android Q

I would like to work on this!

### PR DESCRIPTION
**Pull request by divyank00**
Fixes #1272 

Screenshot 

Desktop_20200119190010

Please make sure these boxes are checked before submitting your pull request - thanks!

- [x] Apply the `AndroidStyle.xml` style template to your code in Android Studio.

- [x] Run the unit tests with `./gradlew check` to make sure you didn't break anything

- [x] If you have multiple commits please combine them into one commit by squashing them.

### CODE REVIEW

**Review by devansh-299 — COMMENTED** [review]
Please add screenshots or recordings for the ui changes and squash your commits

**Comment by divyank00** [comment]
Recording added!

**Comment by divyank00** [comment]
@devansh-299  Recording added

File: app/src/main/java/org/mifos/mobile/ui/fragments/SavingsMakeTransferFragment.java [hunk 1]
```diff
@@ -397,6 +398,10 @@ public void onNothingSelected(AdapterView<?> parent) {
      */
     @OnClick(R.id.btn_pay_to)
     public void payToSelected() {
+        if (payTo == null) {
+            showToaster("Select Account to Pay To");
```
Comment (devansh-299) [inline app/src/main/java/org/mifos/mobile/ui/fragments/SavingsMakeTransferFragment.java:402]: Please convert this to string resource

File: app/src/main/java/org/mifos/mobile/ui/fragments/SavingsMakeTransferFragment.java [hunk 2]
```diff
@@ -414,6 +419,10 @@ public void payToSelected() {
      */
     @OnClick(R.id.btn_pay_from)
     public void payFromSelected() {
+        if (payFrom == null) {
+            showToaster("Select Account to Pay From");
```
Comment (devansh-299) [inline app/src/main/java/org/mifos/mobile/ui/fragments/SavingsMakeTransferFragment.java:423]: Convert this also

**Comment by divyank00** [comment]
Hello, I waned to ask you that how can i try for this years gsoc through
mifos?

**Comment by devansh-299** [comment]
Hi @divyank00 , I think the best way is to contribute to the organization both in terms of code and by helping others. While contributing we get to know more and more about the project details and thus helps us in knowing what actually needs to be done
All the best for your journey :)

**Pull request closed without merging**